keps/sig-scheduling/4832-async-preemption/README.md

```diff
@@ -0,0 +1,410 @@
+This is purely internal feature for kube-scheduler, so no other special actions are required outside the scheduler.
+
+**Downgrade**
+
+Users need to disable the feature gate.
+
+### Version Skew Strategy
+
+This is purely internal feature for kube-scheduler, and hence no version skew strategy.
+
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+- [x] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name: `SchedulerAsyncPreemption`
+  - Components depending on the feature gate:
+- [ ] Other
+  - Describe the mechanism:
+  - Will enabling / disabling the feature require downtime of the control
+    plane?
+  - Will enabling / disabling the feature require downtime or reprovisioning
+    of a node?
+
+###### Does enabling the feature change any default behavior?
+
+No. 
+But, like mentioned in [When kube-apiserver is unstable](#when-kube-apiserver-is-unstable), scheduling results could be different.
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes.
+The feature can be disabled in Alpha and Beta versions
+by restarting kube-scheduler with the feature-gate off.
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+The scheduler again starts to run PostFilter asynchronously.
+
+###### Are there any tests for feature enablement/disablement?
+
+Given it's purely internal feature and enablement/disablement requires restarting the component (to change the value of feature flag), 
+having feature tests is enough.
+
+### Rollout, Upgrade and Rollback Planning
+
+<!--
+This section must be completed when targeting beta to a release.
+-->
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+The partly failure in the rollout isn't there because the scheduler is only the component to rollout this feature. 
+But, if upgrading the scheduler itself fails somehow, new Pods won't be scheduled anymore. 
+(while Pods, which are already scheduled, won't be affected in any cases.)
+
+###### What specific metrics should inform a rollback?
+
+Maybe something goes wrong with the preemption if `goroutines_duration_seconds{operation=preemption}` takes too long time.
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+No. This feature is an internal feature of the scheduler, 
+and just upgrading it and upgrade->downgrade->upgrade are both the same.
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+No.
+
+### Monitoring Requirements
+
+###### How can an operator determine if the feature is in use by workloads?
+
+This feature is used during all Pods' preemption if the feature gate is enabled.
+You can find Pods that have experienced the preemption by referring to `.Status.NominatedNodeName`.
```

Comment:
Those are the ones that cause preemption for other pods.
How about the victims?